package.json

```diff
@@ -84,7 +84,6 @@
     "react-twitter-embed": "^4.0.4",
     "react-youtube-embed": "^1.0.3",
     "reading-time": "^1.5.0",
-    "recharts": "^2.12.0",
```

Comment:
Why recharts library is removed from package.json?

Reply:
Removed it from dependency and installed again as devDependency

Reply:
You have installed types of `recharts` in devDependency. This is the actual library.

Reply:
fixed this